phpBB/styles/prosilver/template/overall_header.html

```diff
@@ -60,7 +60,7 @@
 <!-- IF S_COOKIE_NOTICE -->
 	<link href="{T_ASSETS_PATH}/cookieconsent/cookieconsent.min.css?assets_version={T_ASSETS_VERSION}" rel="stylesheet">
 <!-- ENDIF -->
-
+	<!-- <link href="{T_THEME_PATH}/jquery.atwho.css" rel="stylesheet"> -->
```

Comment:
Did you test that it works correctly in the posting_body.html? If it does, remove it from here :).